floyd/cli/run.py

```diff
@@ -171,6 +176,10 @@ def run(ctx, gpu, env, message, data, mode, open, tensorboard, gpup, cpup, comma
         sys.exit(3)
 
     command_str = ' '.join(command)
+    if command_str and mode in ('jupyter', 'serve'):
+        floyd_logger.error('Command argument "%s" cannot be used with mode: %s', command_str, mode)
```

Comment:
Can we add a docs page as a link for more information? New users might be confused.